_pkgdown.yml

```diff
@@ -78,6 +78,7 @@ reference:
       - eval_capture_conditions
       - print_ard_conditions
       - round5
+      - cards.options
```

Comment:
maybe these two could be together? would it make sense?

Reply:
ah, interesting idea. let's leave as it is for now, and perhaps in the future we can combine.